hi-heavens/content-management-api
# ch(connect-db): connect to the db

### What does this PR do?
* To successfully create a connection with the contentdb

### Description of the task to be completed?
* Install the dependencies 
* Provide the necessary db parameters to connect successfully

### How should this be manually tested?
* On the terminal, run the command npm run start:dev to start the server and confirm DB connection
* No error returned if successful
